homeassistant/components/ebusd/const.py

```diff
@@ -191,7 +191,7 @@
             4,
             SensorDeviceClass.TEMPERATURE,
         ],
-        "WaterPreasure": ["WaterPressure", PRESSURE_BAR, "mdi:pipe", 4, None],
+        "WaterPressure": ["WaterPressure", PRESSURE_BAR, "mdi:pipe", 4, None],
```

Comment:
I don't think this can be changed?
Did you test this code change?

Reply:
Looking at the docs this is indeed a typo, however since we don't know if users may have already use the sensor name with the typo I think this should be marked as a breaking change. 
https://www.home-assistant.io/integrations/ebusd/#waterpressure
Also since I only looked at code, I think it should also be validated by someone testing it.

Reply:
This was added in #26750 by @sashao, perhaps they could check?

Reply:
@thecode is right, this changes end-user configuration and thus a breaking change.
This affects monitored conditions, which is [ADR-0003](https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md); we should no longer allow and use, thus most be refactored.

Additionally, [ADR-0010](https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md) applies, which doesn't allow further changes to YAML configuration (which this change does make).